templates/node/index.d.ts.twig

```diff
@@ -131,11 +131,13 @@ declare module "{{ language.params.npmPackage|caseDash }}" {
 
     static fromBuffer(buffer: Buffer, filename: string): InputFile;
 
-    static fromBlob(blob: Blob, filename: string): InputFile;
+    static fromBlob(blob: buffer.Blob, filename: string): Promise<InputFile>;
```

Comment:
Above we do `buffer: Buffer`, so here I would expect `blob: Buffer.Blob` (with uppercase B). Is it lowercase here on purpose?

Reply:
No type complaints when using `buffer`. Also `Buffer` is the class while `buffer` is the module.